### ISSUE TITLE
Add stars to solutions of starred exercises

### ISSUE THREAD

**Issue by wojtask** [wojtask/clrs4e-solutions#1578]
Find the most appealing way, so that it won't break spacing of exercise labels.

**Comment by wojtask** [comment]
Stars are common in regular exercises. Whole problems are not starred and there are only a few (3?) starred subproblems in the whole book.

**Comment by wojtask** [comment]
There are even whole starred sections, like C.5. The star is just before the section number, it also shows up in toc.

**Comment by wojtask** [comment]
Stars now appear next to solutions to starred exercises and subproblems. To make room for them, I had to slightly change spacing of the margin labels, for exercises and problems. Stars can also be added to whole sections, thanks to additional list parameter in `\MakeChapter`. However, there are two caveats:
* In order to place the stars aligned on the margin, I used magic constants determined "by hand", and so it'd best to find a better way to achieve the same result. I opened #1582 to address that.
* We still need to show the starred sections in table of contents. #1580 will address that.
